tool-plugins/intellij/build.gradle

```diff
@@ -52,8 +52,9 @@ sourceSets {
 apply plugin: 'org.jetbrains.intellij'
 intellij {
     // For a full list of IntelliJ IDEA releases, please see https://www.jetbrains.com/intellij-repository/releases.
+
     version ideaVersion
-    type ideaType
```

Comment:
Shall we remove `ideaType` from `gradle.properties` as well?

Reply:
Removed